src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -264,7 +263,12 @@ public void Properties_AddItemToDictionary_ItemPresent()
             }
         }
 
-        [ActiveIssue(20010, TargetFrameworkMonikers.Uap)]
+        // ArgumentException: The given System.Uri cannot be converted into a Windows.Foundation.Uri.
+        // Please see http://go.microsoft.com/fwlink/?LinkID=215849 for details.
+        //
+        // This is the URI that fails the test:
+        //     LoopbackServer.GetIPv6LinkLocalAddress() == http://[fe80::3df1:c851:8775:4c5b]:25545/
```

Comment:
It looks like a bug in Windows.Foundation.Uri or possible the magic interop layer between System.Uri and Windows.Foundation.Uri.  I was planning on opening up another issue and changing the issue here number from the epic.  I'll do that in the next PR of changes to these tests.